### DISCUSSION TITLE
use of npm package

### DISCUSSION THREAD

**Discussion by devper-cloud** [quasarframework/quasar#10501] [General - Components / Directives / etc]
I try to import the platform sdk in a quasar application but so far I always get errors.

this is my code:

const platformClient = require('purecloud-platform-client-v2')
const client = platformClient.ApiClient.instance
console.log(client);
Error:

Uncaught (in promise) TypeError: Cannot read property 'instance' of undefined

when I try to do it like:

const platformClient = require('purecloud-platform-client-v2/dist/node/purecloud-platform-client-v2')
const client = platformClient.ApiClient.instance
console.log(client);

I get these errors:

App • DONE • "UI" compiled with errors • 548ms

App • ERROR • UI in ./node_modules/configparser/node_modules/mkdirp/index.js

Module not found: Can't resolve imported dependency "fs"
Did you forget to install it? You can run: npm install --save fs

App • ERROR • UI in ./node_modules/configparser/src/configparser.js

Module not found: Can't resolve imported dependency "fs"
Did you forget to install it? You can run: npm install --save fs

App • ERROR • UI in ./node_modules/purecloud-platform-client-v2/dist/node/purecloud-platform-client-v2.js

Module not found: Can't resolve imported dependency "fs"
Did you forget to install it? You can run: npm install --save fs

App • ERROR • UI in ./node_modules/winston/dist/winston/tail-file.js

Module not found: Can't resolve imported dependency "fs"
Did you forget to install it? You can run: npm install --save fs

App • ERROR • UI in ./node_modules/winston/dist/winston/transports/file.js

Module not found: Can't resolve imported dependency "fs"
Did you forget to install it? You can run: npm install --save fs

App • ERROR • UI in ./node_modules/winston/dist/winston/transports/http.js

Module not found: Can't resolve imported dependency "http"
Did you forget to install it? You can run: npm install --save http

App • ERROR • UI in ./node_modules/winston/dist/winston/transports/http.js

Module not found: Can't resolve imported dependency "https"
Did you forget to install it? You can run: npm install --save https

App • ERROR • UI in ./node_modules/zlib/lib/zlib.js

Module not found: Can't resolve imported dependency "./zlib_bindings"

App • COMPILATION FAILED • Please check the log above for details.

If I run this in node js file it works.
What am I doing wrong?

**Comment by ontwikkelfabriek** [comment]
@devper-cloud 

It looks like the `purecloud-platform-client-v2` is a nodejs app instead of a web library that can run in the browser/Quasar.
